setup.py

```diff
@@ -35,12 +36,11 @@ def read(fname):
 
 
 def in_scriptdir(path):
-    return os.path.join(SCRIPTDIR, os.path.normpath(path))
+    return join(SCRIPTDIR, normpath(path))
 
 
 def rebuild_catalogs():
-    import subprocess
-    subprocess.call('python scripts/cmpmsgs.py librarian/locales', shell=True)
+    call('python scripts/cmpmsgs.py librarian/locales', shell=True)
```

Comment:
to collect messages: `librarian --conf librarian.ini --xmsgs`
to compile messages: `librarian --conf librarian.ini --cmsgs`